### PR TITLE
Create script.js

### PR DESCRIPTION
**Pull request by RhyenSftwareEngr**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (grgn2002) [inline script.js:1]: This code is confusing. Change

Comment (AVILLANOZA) [inline script.js:1]: add nodejs

Comment (RhyenSftwareEngr) [reply script.js:1]: Yeah I agree

Comment (AVILLANOZA) [inline script.js:1]: very good

**Review by AVILLANOZA — APPROVED** [review]

**Pull request merged**